src/Main.java

```diff
@@ -0,0 +1,53 @@
+import java.util.Arrays;
+import java.util.Collection;
+import java.util.List;
+
+public class Main {
+
+    public static void main(String[] args){
+        List<Integer> firstList = Arrays.asList(1, 2, 3, 4, 5, 6, 7, 8);
+        List<Integer> secondList = Arrays.asList(11, 12, 13, 14, 15, 16, 17,18);
+
+        countingEvenNumber(secondList);
+        replaceElementInFirstList(firstList, secondList, 3);
+        replaceElements(firstList, secondList, 0);
+        findMax(secondList);
+    }
+    //utility method, which counts even numbers in given collection
+    public static <T extends Number> void countingEvenNumber(List<T> list){
+        int count = 0;
+        System.out.print("Even numbers - ");
+        for(T t : list){
+            if(t.intValue() % 2 == 0){
+                count = count + t.intValue();
+                System.out.print(t +" ");
+            }
+        }
+        System.out.println("Summ of even numbers = " + count);
```

Comment:
The goal was to check if you understood generics right. And it seems you did. No questions here.
But don't forget about the task: count is not a sum :)